components/header/header.tsx

```diff
@@ -29,7 +29,7 @@ export default function Header() {
           <a href="#features" className={style.link}>
             Features
           </a>
-          <a href="#" className={style.link}>
+          <a href="#get-started" className={style.link}>
```

Comment:
nice one! I'm glad you remembered 👍 

I think that completes the missing link for the header? If so we could perhaps remove the comment.

Reply:
Oh good call. I'll remove before merging. Thanks for flagging :)